pom.xml

```diff
@@ -0,0 +1,96 @@
+ ~ in the Software without restriction, including without limitation the rights
+ ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ ~ copies of the Software, and to permit persons to whom the Software is
+ ~ furnished to do so, subject to the following conditions:
+ ~
+ ~ The above copyright notice and this permission notice shall be included in
+ ~ all copies or substantial portions of the Software.
+ ~
+ ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ ~ THE SOFTWARE.
+ -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>2.9</version>
+  </parent>
+
+  <artifactId>bouncycastle-api</artifactId>
+  <version>0.1-SNAPSHOT</version>
+  <packaging>hpi</packaging>
+
+  <name>bouncycastle API Plugin</name>
+  <description>
+    This plugin provides an stable API to Bouncy Castle related tasks.
+  </description>
+  <url>http://wiki.jenkins-ci.org/display/JENKINS/Bouncy+Castle+API+Plugin</url>
+  <licenses>
+    <license>
+      <name>The MIT license</name>
+      <url>http://www.opensource.org/licenses/mit-license.php</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>alvarolobato</id>
+      <name>Alvaro Lobato</name>
+      <email>alobato@cloudbees.com</email>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git://github.com/jenkinsci/bouncycastle-api.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/bouncycastle-api.git</developerConnection>
+    <url>http://github.com/jenkinsci/bouncycastle-api</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <jenkins.version>1.609</jenkins.version>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
+    </repository>
+  </repositories>
+
+  <pluginRepositories>
+    <pluginRepository>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <version>1.47</version>
```

Comment:
Yes, changing it.
El 6/6/2016 13:39, "Antonio" <notifications@github.com> escribió:

> In pom.xml
> <https://github.com/jenkinsci/bouncycastle-api/pull/1#discussion_r65875883>
> :
>
> > +      <url>https://repo.jenkins-ci.org/public/</url>
> > +    </repository>
> > +  </repositories>
> > +
> > +  <pluginRepositories>
> > +    <pluginRepository>
> > +      <id>repo.jenkins-ci.org</id>
> > +      <url>https://repo.jenkins-ci.org/public/</url>
> > +    </pluginRepository>
> > +  </pluginRepositories>
> > +
> > +  <dependencies>
> > +    <dependency>
> > +      <groupId>org.bouncycastle</groupId>
> > +      <artifactId>bcpkix-jdk15on</artifactId>
> > +      <version>1.47</version>
>
> Then it must be <scope>provided</scope>?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jenkinsci/bouncycastle-api/pull/1/files/e3a0c8fb8b6a9b15c38033d17f9e17b91a893960#r65875883>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQXNBBzc6zwfUKgDNtG2ZjBN4GjFlANqks5qJAbhgaJpZM4Iuv42>
> .
>